frontend/packages/llama-stack-modular-ui/frontend/src/app/Chatbot/sourceUpload/ChatbotSourceUploadPanel.tsx

```diff
@@ -22,57 +20,53 @@ type ChatbotSourceUploadPanelProps = {
 };
 
 const ChatbotSourceUploadPanel: React.FC<ChatbotSourceUploadPanelProps> = ({
-  alert,
+  successAlert,
+  errorAlert,
   handleSourceDrop,
   selectedSource,
   selectedSourceSettings,
   removeUploadedSource,
   setSelectedSourceSettings,
 }) => (
-  <DrawerPanelContent isResizable defaultSize="400px" minSize="300px">
+  <>
     <AlertGroup hasAnimations isToast isLiveRegion>
-      {alert}
+      {successAlert}
+      {errorAlert}
     </AlertGroup>
-    <DrawerPanelBody>
-      <Title
-        headingLevel="h5"
-        style={{ fontWeight: 'bold', marginBottom: 'var(--pf-t--global--spacer--2xl)' }}
-      >
-        Sources
-      </Title>
-      <MultipleFileUpload
-        onFileDrop={handleSourceDrop}
-        dropzoneProps={{
-          accept: {
-            'application/msword': ['.doc'],
-            'application/pdf': ['.pdf'],
-          },
-          maxFiles: 1,
-        }}
-        aria-label="Source upload area"
-      >
-        <MultipleFileUploadMain
-          browseButtonText="Add"
-          titleIcon={<FileIcon />}
-          titleText="Drag and drop file here"
-          titleTextSeparator="or"
-          infoText="Upload a PDF or DOC file to be used in retrieval."
-        />
-        {selectedSourceSettings &&
-          selectedSource.map((file) => (
-            <MultipleFileUploadStatusItem
-              file={file}
-              key={file.name}
-              onClearClick={() => {
-                removeUploadedSource(file.name);
-                setSelectedSourceSettings(null);
-              }}
-              aria-label={`Uploaded file: ${file.name}`}
-            />
-          ))}
-      </MultipleFileUpload>
-    </DrawerPanelBody>
-  </DrawerPanelContent>
+    <MultipleFileUpload
+      onFileDrop={handleSourceDrop}
+      dropzoneProps={{
+        accept: {
+          'application/pdf': ['.pdf'],
+          'application/msword': ['.doc'],
+          'text/csv': ['.csv'],
+        },
+        maxFiles: 1,
```

Comment:
can we extract the value of `maxFiles` into a constant and set it to a value greater than 1 ? That would:
- improve user experience
- test or exhibit our ability to do the multiple file uploads, including on the BFF side ?

Reply:
As a follow up (not urgent) - Do we also need to limit on the size? Or add a validation for it on the UI side. 
On LS end - there is no size limitation though, but not sure if we need to do something on our end given we are using inline milvus.

Reply:
@varshaprasad96 we should have validation from UI to prevent extremely large file upload
Typically, most of the other component in UI has 1mb as maxSize, I can follow that